iOS/ARShooterExample/ARShooterGame/ARShooter/LoginViewController/LoginViewController.swift

```diff
@@ -194,7 +194,7 @@ class LoginViewController: UIViewController {
             if appPortsDict.capacity == 0 {
                 throw LoginViewControllerError.runtimeError("No AppPorts in dictionary")
             }
-            // Select AppPort corresponding to internal port 3001
+            // Select AppPort corresponding to internal port 1337
```

Comment:
Thanks for the fix

Reply:
When did we switch to this elite port number?

Reply:
I had been using it. the 3001 was a mistake. i have no idea why i used 1337

Reply:
https://en.wikipedia.org/wiki/Leet

Reply:
I can change the port and redeploy

Reply:
I'm not saying there's anything wrong with that port number. I just thought it was funny that you chose it, and assumed you had done so intentionally, hence my "elite port number" comment.